File: Iniciante/1046/1046.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    String[] values = scanner.nextLine().split(" ");

    int h1 = Integer.parseInt(values[0]);
    int h2 = Integer.parseInt(values[1]);

    if (h1 >= h2) {
      System.out.printf("O JOGO DUROU %d HORA(S)\n", 24 - (h1 - h2));
    } else {
      System.out.printf("O JOGO DUROU %d HORA(S)\n", h2 - h1);
    }

  }
}

File: Iniciante/1046/1046.rs
use std::io;

fn main() {
  let mut input = String::new();

  io::stdin().read_line(&mut input).unwrap();

  let values: Vec<u16> = input.split_whitespace()
    .map(|x| x.parse().unwrap())
    .collect();

  let h1 = values[0];
  let h2 = values[1];

  if h1 >= h2 {
    println!("O JOGO DUROU {} HORA(S)", 24 - (h1 - h2));
  } else {
    println!("O JOGO DUROU {} HORA(S)", h2 - h1);
  }
}
